tracing-forest/src/printer/pretty.rs

```diff
@@ -225,13 +225,16 @@ struct ColorLevel(Level);
 
 impl fmt::Display for ColorLevel {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match self.0 {
-            Level::TRACE => Color::Purple.bold().paint("TRACE   "),
-            Level::DEBUG => Color::Blue.bold().paint("DEBUG   "),
-            Level::INFO => Color::Green.bold().paint("INFO    "),
-            Level::WARN => Color::RGB(252, 234, 160).bold().paint("WARN    "), // orange
-            Level::ERROR => Color::Red.bold().paint("ERROR   "),
-        }
-        .fmt(f)
+        let (color, text) = match self.0 {
+            Level::TRACE => (Color::Purple, "TRACE"),
+            Level::DEBUG => (Color::Blue, "DEBUG"),
+            Level::INFO => (Color::Green, "INFO"),
+            Level::WARN => (Color::RGB(252, 234, 160), "WARN"), // orange
+            Level::ERROR => (Color::Red, "ERROR"),
+        };
+        let style = color.bold();
+        write!(f, "{}", style.prefix())?;
+        f.pad(text)?;
+        write!(f, "{}", style.suffix())
```

Comment:
pedantic: I forgot that [`Level::as_str`](https://docs.rs/tracing/latest/tracing/struct.Level.html#method.as_str) exists, your choice though
```suggestion
        let color = match self.0 {
            Level::TRACE => Color::Purple,
            Level::DEBUG => Color::Blue,
            Level::INFO => Color::Green,
            Level::WARN => Color::RGB(252, 234, 160), // orange
            Level::ERROR => Color::Red,
        };
        let style = color.bold();
        write!(f, "{}", style.prefix())?;
        f.pad(self.0.as_str())?;
        write!(f, "{}", style.suffix())
```